documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1545,6 +1531,56 @@ By default, no operations are skipped.
 
 |===
 
+include::{partialsdir}/modules/all-connectors/ref-connector-configuration-database-history-properties.adoc[leveloffset=+1]
+
+////
+[id="{context}-pass-through-configuration-properties-for-database-drivers"]
+==== Pass-through configuration properties for database drivers
+
+The {prodname} connector provides for pass-through configuration of the database driver.
+Pass-through database properties begin with the prefix `database.*`.
+For example, the property `database.tinyInt1isBit=false` is passed to the JDBC URL.
```

Comment:
tinyInt1isBit is specific to MySQL. Also, why isn't this part of a partial? I think, you'd want to have two partials:

* One for the DB pass through props
* One for the DB history stuff

Reply:
Same comment as earlier about commented out section. 

I already added the passthrough content to the DB history partial. I can split it out again into its own partial, but I'm not sure that's really necessary. WDYT?